[UPDATE] POST /image 에 withCredentials 옵션 추가

pje1740 opened the pull request:
[comment]: <> (PR 태그를 명시해주세요. [ADD] / [UPDATE] / [BUG])

### 요약
POST /image 에  withCredentials 옵션 추가

### 핵심 로직 및 내용

### 기타 참고 사항

pje1740 committed:
[FIX] add withCredentials option to image api.

[why]
This is needed to connect with the back-end server.

[what]
POST /images

[etc]

thesulks reviewed (approved):
LGTM 👍 
사소한 것이긴 한데, PR 태그를 대문자로 통일(`UPDATE`)하는 게 좋을 것 같습니다

pje1740 committed:
[FIX] fix axios parameter style.

[why]
It is possible to use axios.post and pass headers as parameters.

[what]
axios.post

[etc]

JeesooHaa reviewed (approved):
수정 후가 통일성 있어서 좋네요 👍
